### ISSUE TITLE
Julesjordan not pulling metadata

### ISSUE THREAD

**Issue by xmxier** [PAhelper/PhoenixAdult.bundle#193]
Julesjordan
name of video : Keisha Grey Natural Busty Teen Milks Manuels’s Cock

**Comment by chempke** [comment]
I can confirm this as a bug

```
2019-06-11 21:14:11,759 (4478) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjI0NjY1NXMyCmlkczQ0Cmp1bGVzSm9yZGFuIC0gS2Vpc2hhIEdyZXkgTmF0dXJhbCBCdXN0eSBUZWVuczQKbmFtZXM0CjIwMTlzNAp5ZWFycjAK
2019-06-11 21:14:11,759 (4478) :  DEBUG (runtime:49) - Received packed state data (2776 bytes)
2019-06-11 21:14:11,762 (4478) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjI0NjY1NXMyCmlkczQ0Cmp1bGVzSm9yZGFuIC0gS2Vpc2hhIEdyZXkgTmF0dXJhbCBCdXN0eSBUZWVuczQKbmFtZXM0CjIwMTlzNAp5ZWFycjAK
2019-06-11 21:14:11,762 (4478) :  INFO (agentkit:957) - Searching for matches for {'year': '2019', 'id': '246655', 'name': 'julesJordan - Keisha Grey Natural Busty Teen'}
2019-06-11 21:14:11,762 (4478) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/246655/tree'
2019-06-11 21:14:11,766 (4478) :  INFO (logkit:16) - *******MEDIA TITLE****** julesJordan - Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,766 (4478) :  INFO (logkit:16) - Getting Search Settings for: julesJordan - Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,767 (4478) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation: julesJordan Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,770 (4478) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: julesJordan Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - ^^^^^^^ siteID: 432
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - julesjordan  vs jules jordan
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - 2
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - 3
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - 6
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - searchTitle (before date processing): Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - Search Title: Keisha Grey Natural Busty Teen
2019-06-11 21:14:11,773 (4478) :  INFO (logkit:16) - Keisha%20Grey%20Natural%20Busty%20Teen
2019-06-11 21:14:11,779 (4478) :  DEBUG (networking:166) - Requesting 'https://www.julesjordan.com/trial/search.php?query=Keisha%20Grey%20Natural%20Busty%20Teen'
2019-06-11 21:14:12,292 (4478) :  INFO (logkit:16) - releaseDate: 2018-10-28
2019-06-11 21:14:12,296 (4478) :  DEBUG (runtime:88) - Sending packed state data (2808 bytes)
2019-06-11 21:14:12,296 (4478) :  DEBUG (runtime:924) - Response: [200] str, 850 bytes
2019-06-11 21:14:13,210 (3a5c) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyNDY2NTVzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzMTQ0CmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly9odHRwczpfX3d3dy5qdWxlc2pvcmRhbi5jb21fdHJpYWxfc2NlbmVzX0tlaXNoYS1HcmV5LU5hdHVyYWwtQmlnLVRpdHMtQ3Vtc2hvdC1UZWVuLUJ1c3R5X3ZpZHMuaHRtbHw0MzI%40bGFuZz1lbnM0Cmd1aWRzMTAyCmh0dHBzOl9fd3d3Lmp1bGVzam9yZGFuLmNvbV90cmlhbF9zY2VuZXNfS2Vpc2hhLUdyZXktTmF0dXJhbC1CaWctVGl0cy1DdW1zaG90LVRlZW4tQnVzdHlfdmlkcy5odG1sfDQzMnMyCmlkcjAK
2019-06-11 21:14:13,210 (3a5c) :  DEBUG (runtime:49) - Received packed state data (7220 bytes)
2019-06-11 21:14:13,211 (3a5c) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyNDY2NTVzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzMTQ0CmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly9odHRwczpfX3d3dy5qdWxlc2pvcmRhbi5jb21fdHJpYWxfc2NlbmVzX0tlaXNoYS1HcmV5LU5hdHVyYWwtQmlnLVRpdHMtQ3Vtc2hvdC1UZWVuLUJ1c3R5X3ZpZHMuaHRtbHw0MzI@bGFuZz1lbnM0Cmd1aWRzMTAyCmh0dHBzOl9fd3d3Lmp1bGVzam9yZGFuLmNvbV90cmlhbF9zY2VuZXNfS2Vpc2hhLUdyZXktTmF0dXJhbC1CaWctVGl0cy1DdW1zaG90LVRlZW4tQnVzdHlfdmlkcy5odG1sfDQzMnMyCmlkcjAK
2019-06-11 21:14:13,213 (3a5c) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://https:__www.julesjordan.com_trial_scenes_Keisha-Grey-Natural-Big-Tits-Cumshot-Teen-Busty_vids.html|432?lang=en
2019-06-11 21:14:13,213 (3a5c) :  DEBUG (model:234) - Deserializing from C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\6\9430502f094b92672ddb0cabb8df8e10ce3dcea.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-11 21:14:13,216 (3a5c) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/246655/tree'
2019-06-11 21:14:13,232 (3a5c) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-11 21:14:13,233 (3a5c) :  INFO (logkit:16) - 432
2019-06-11 21:14:13,233 (3a5c) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-11 21:14:13,237 (3a5c) :  DEBUG (networking:166) - Requesting 'https://www.julesjordan.com/trial/scenes/Keisha-Grey-Natural-Big-Tits-Cumshot-Teen-Busty_vids.html'
2019-06-11 21:14:13,756 (3a5c) :  CRITICAL (core:574) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://https:__www.julesjordan.com_trial_scenes_Keisha-Grey-Natural-Big-Tits-Cumshot-Teen-Busty_vids.html|432?lang=en' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 1079, in update
    metadata = PAsearchSites.siteJulesJordan.update(metadata,siteID,movieGenres,movieActors)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\siteJulesJordan.py", line 36, in update
    metadata.summary = detailsPageElements.xpath('//span[@class="update_dvds"]/a')[0].text_content().strip()
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2019-06-11 21:14:13,757 (3a5c) :  DEBUG (model:229) - Serializing to C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\6\9430502f094b92672ddb0cabb8df8e10ce3dcea.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-11 21:14:13,763 (3a5c) :  DEBUG (runtime:88) - Sending packed state data (7252 bytes)
2019-06-11 21:14:13,763 (3a5c) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes


```

**Comment by claygoldfinch** [comment]
Thanks for the log! Is this happening in multiple JJ scenes, or is this the only one?

**Comment by chempke** [comment]
I found a 2nd one and tried over 100 scenes so far.

[JulesJordan - Evelyn Claire Takes On Dredd The BBC Monster](https://www.julesjordan.com/trial/scenes/Evelyn-Claire-Interracial-Dredd-Big-Black-Cock-4k-Facial_vids.html)


```
2019-06-12 19:33:35,262 (1670) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo0N2NjYzBkZDI2OGIwNzI3czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczU2Ckp1bGVzam9yZGFuIEV2ZWx5biBDbGFpcmUgVGFrZXMgb24gRHJlZGQgdGhlIEJiYyBNb25zdGVyczQKbmFtZXMxOTIKSyUzQSU1Q0NvbnRlbnQlNUNKdWxlc0pvcmRhbiU1Q0p1bGVzSm9yZGFuJTIwLSUyMEV2ZWx5biUyMENsYWlyZSUyMFRha2VzJTIwT24lMjBEcmVkZCUyMFRoZSUyMEJCQyUyME1vbnN0ZXIlNUNKdWxlc0pvcmRhbiUyMC0lMjBFdmVseW4lMjBDbGFpcmUlMjBUYWtlcyUyME9uJTIwRHJlZGQlMjBUaGUlMjBCQkMlMjBNb25zdGVyJTJFbXA0czgKZmlsZW5hbWVzNDAKMTVkMDQ5YzIzZWQ1MDE1MDcwMjBhYjAwYTMyZjY1Mzk2Y2I3NzY1YnM4CnBsZXhIYXNoczcKMTQyMDIyNHM4CmR1cmF0aW9uczYKMjk0OTUyczIKaWRyMAo_
2019-06-12 19:33:35,263 (1670) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2019-06-12 19:33:35,263 (1670) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo0N2NjYzBkZDI2OGIwNzI3czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczU2Ckp1bGVzam9yZGFuIEV2ZWx5biBDbGFpcmUgVGFrZXMgb24gRHJlZGQgdGhlIEJiYyBNb25zdGVyczQKbmFtZXMxOTIKSyUzQSU1Q0NvbnRlbnQlNUNKdWxlc0pvcmRhbiU1Q0p1bGVzSm9yZGFuJTIwLSUyMEV2ZWx5biUyMENsYWlyZSUyMFRha2VzJTIwT24lMjBEcmVkZCUyMFRoZSUyMEJCQyUyME1vbnN0ZXIlNUNKdWxlc0pvcmRhbiUyMC0lMjBFdmVseW4lMjBDbGFpcmUlMjBUYWtlcyUyME9uJTIwRHJlZGQlMjBUaGUlMjBCQkMlMjBNb25zdGVyJTJFbXA0czgKZmlsZW5hbWVzNDAKMTVkMDQ5YzIzZWQ1MDE1MDcwMjBhYjAwYTMyZjY1Mzk2Y2I3NzY1YnM4CnBsZXhIYXNoczcKMTQyMDIyNHM4CmR1cmF0aW9uczYKMjk0OTUyczIKaWRyMAo_
2019-06-12 19:33:35,263 (1670) :  INFO (agentkit:957) - Searching for matches for {'openSubtitlesHash': '47ccc0dd268b0727', 'name': 'Julesjordan Evelyn Claire Takes on Dredd the Bbc Monster', 'filename': 'K%3A%5CContent%5CJulesJordan%5CJulesJordan%20-%20Evelyn%20Claire%20Takes%20On%20Dredd%20The%20BBC%20Monster%5CJulesJordan%20-%20Evelyn%20Claire%20Takes%20On%20Dredd%20The%20BBC%20Monster%2Emp4', 'plexHash': '15d049c23ed501507020ab00a32f65396cb7765b', 'duration': '1420224', 'id': '294952'}
2019-06-12 19:33:35,265 (1670) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294952/tree'
2019-06-12 19:33:35,269 (1670) :  INFO (logkit:16) - *******MEDIA TITLE****** Julesjordan Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,269 (1670) :  INFO (logkit:16) - Getting Search Settings for: Julesjordan Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,269 (1670) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation: Julesjordan Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,273 (1670) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Julesjordan Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,275 (1670) :  INFO (logkit:16) - ^^^^^^^ siteID: 432
2019-06-12 19:33:35,275 (1670) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2019-06-12 19:33:35,275 (1670) :  INFO (logkit:16) - julesjordan  vs jules jordan
2019-06-12 19:33:35,275 (1670) :  INFO (logkit:16) - 2
2019-06-12 19:33:35,276 (1670) :  INFO (logkit:16) - 3
2019-06-12 19:33:35,276 (1670) :  INFO (logkit:16) - 6
2019-06-12 19:33:35,276 (1670) :  INFO (logkit:16) - searchTitle (before date processing): Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,276 (1670) :  INFO (logkit:16) - Search Title: Evelyn Claire Takes on Dredd the Bbc Monster
2019-06-12 19:33:35,276 (1670) :  INFO (logkit:16) - Evelyn%20Claire%20Takes%20on%20Dredd%20the%20Bbc%20Monster
2019-06-12 19:33:35,283 (1670) :  DEBUG (networking:166) - Requesting 'https://www.julesjordan.com/trial/search.php?query=Evelyn%20Claire%20Takes%20on%20Dredd%20the%20Bbc%20Monster'
2019-06-12 19:33:35,865 (1670) :  INFO (logkit:16) - releaseDate: 2019-03-30
2019-06-12 19:33:35,868 (1670) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2019-06-12 19:33:35,868 (1670) :  DEBUG (runtime:924) - Response: [200] str, 848 bytes
2019-06-12 19:33:38,519 (1748) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ5NTJzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzMTUzCmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly9odHRwczpfX3d3dy5qdWxlc2pvcmRhbi5jb21fdHJpYWxfc2NlbmVzX0V2ZWx5bi1DbGFpcmUtSW50ZXJyYWNpYWwtRHJlZGQtQmlnLUJsYWNrLUNvY2stNGstRmFjaWFsX3ZpZHMuaHRtbHw0MzI%40bGFuZz1lbnM0Cmd1aWRzMTExCmh0dHBzOl9fd3d3Lmp1bGVzam9yZGFuLmNvbV90cmlhbF9zY2VuZXNfRXZlbHluLUNsYWlyZS1JbnRlcnJhY2lhbC1EcmVkZC1CaWctQmxhY2stQ29jay00ay1GYWNpYWxfdmlkcy5odG1sfDQzMnMyCmlkcjAK
2019-06-12 19:33:38,519 (1748) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2019-06-12 19:33:38,520 (1748) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ5NTJzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzMTUzCmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly9odHRwczpfX3d3dy5qdWxlc2pvcmRhbi5jb21fdHJpYWxfc2NlbmVzX0V2ZWx5bi1DbGFpcmUtSW50ZXJyYWNpYWwtRHJlZGQtQmlnLUJsYWNrLUNvY2stNGstRmFjaWFsX3ZpZHMuaHRtbHw0MzI@bGFuZz1lbnM0Cmd1aWRzMTExCmh0dHBzOl9fd3d3Lmp1bGVzam9yZGFuLmNvbV90cmlhbF9zY2VuZXNfRXZlbHluLUNsYWlyZS1JbnRlcnJhY2lhbC1EcmVkZC1CaWctQmxhY2stQ29jay00ay1GYWNpYWxfdmlkcy5odG1sfDQzMnMyCmlkcjAK
2019-06-12 19:33:38,520 (1748) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://https:__www.julesjordan.com_trial_scenes_Evelyn-Claire-Interracial-Dredd-Big-Black-Cock-4k-Facial_vids.html|432?lang=en
2019-06-12 19:33:38,522 (1748) :  DEBUG (model:234) - Deserializing from C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\b\f1ee3b08dcb338ced93b9a2f225acec040f6bb1.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-12 19:33:38,523 (1748) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294952/tree'
2019-06-12 19:33:38,539 (1748) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-12 19:33:38,539 (1748) :  INFO (logkit:16) - 432
2019-06-12 19:33:38,539 (1748) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-12 19:33:38,546 (1748) :  DEBUG (networking:166) - Requesting 'https://www.julesjordan.com/trial/scenes/Evelyn-Claire-Interracial-Dredd-Big-Black-Cock-4k-Facial_vids.html'
2019-06-12 19:33:39,223 (1748) :  CRITICAL (core:574) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://https:__www.julesjordan.com_trial_scenes_Evelyn-Claire-Interracial-Dredd-Big-Black-Cock-4k-Facial_vids.html|432?lang=en' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-147963d87\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 1079, in update
    metadata = PAsearchSites.siteJulesJordan.update(metadata,siteID,movieGenres,movieActors)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\siteJulesJordan.py", line 36, in update
    metadata.summary = detailsPageElements.xpath('//span[@class="update_dvds"]/a')[0].text_content().strip()
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-147963d87\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2019-06-12 19:33:39,223 (1748) :  DEBUG (model:229) - Serializing to C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\b\f1ee3b08dcb338ced93b9a2f225acec040f6bb1.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-12 19:33:39,227 (1748) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2019-06-12 19:33:39,227 (1748) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes

```

**Comment by claygoldfinch** [comment]
@xmxier @chempke Alright, should be resolved in PR #217.

You may want to refresh the metadata on any other JJ scenes you have, as my fix resolved issues with both scene summaries and scene collections for all JJ scenes.
